src/views/AdminActivityEdit.vue

```diff
@@ -138,12 +289,23 @@
 <script>
 import { db } from "@/db.js";
 import { mapGetters } from "vuex";
+import Markdown from "vue-markdown";
 
 export default {
-  props: ["slug", "activityName"],
+  components: {Markdown},
+  props: ["slug", "activityName","currentUserSettings"],
```

Comment:
I'm inclined toward `currentUserSettings` coming directly from vuex instead of being passed in; though maybe it doesn't matter because the user doesn't see/change them? What are your thoughts?

Reply:
I thought I couldn't get it to work at first, but now it does. Fixed!